src/components/radios/examples/radios-borderless/index.njk

```diff
@@ -0,0 +1,40 @@
+{% from "components/radios/_macro.njk" import onsRadios %}
+{% from "components/question/_macro.njk" import onsQuestion %}
+
+{% call onsQuestion({
+    "title": "What type of accommodation do you live in?",
+    "classes": "u-mt-no",
+    "legendIsQuestionTitle": true
+}) %}
+{{
+        onsRadios({
+            "name": "property-type",
+            "dontWrap": true,
+            "containerClasses": "u-mb-s",
+            "blockClasses": "radio--no-border",
```

Comment:
I would change this to a single parameter `borderless: true`. The macro would contain the logic to handle which  classes to add i.e. `{% if borderless === true %} u-mb-s ...` etc

Reply:
Interestingly, this is how I initially developed the feature but wasn't too sure on how much control the user would ordinarily have. I will refactor to use a bool.